kubernetes/cray-tapms-operator/Chart.yaml

```diff
@@ -21,9 +21,12 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
+#
+# Remember to change version in conjuction with .version and values.yaml:
+#
 apiVersion: v2
 name: cray-tapms-operator
-version: 0.0.10
+version: 0.0.11
```

Comment:
This file should also show the artifact

  artifacthub.io/images: |-
    - name: cray-tapms-operator
      image: artifactory.algol60.net/csm-docker/stable/cray-tapms-operator:v0.0.11

Reply:
good catch -- fixing

Reply:
fixed